pages/checkout/index.js

```diff
@@ -449,7 +449,7 @@ class CheckoutPage extends Component {
                     )
                   })}
                 </div>
-                <form className="d-flex py-3 borderbottom border-color-gray400">
+                <form className="py-3 borderbottom border-color-gray400 discount">
```

Comment:
Think creating an additional discount class just to apply `display: flex` is redundant here when `d-flex` is available already providing that.
```suggestion
                <form className="d-flex py-3 borderbottom border-color-gray400">
```

Reply:
yes originally had d-flex and d-sm-block but messed by the flex alignment on the input and button. i targeted in css to fix the display block bug on mobile view